.travis.yml

```diff
@@ -44,34 +32,43 @@ before_install:
   - if [ "$GITHUB_COMPOSER_AUTH" ]; then composer config -g github-oauth.github.com $GITHUB_COMPOSER_AUTH; fi
 
 before_script:
-  - if [ "$TYPO3_VERSION" == "^6.2.0" ]; then composer require typo3/cms-composer-installers="1.2.5 as 1.1.4" --no-update; fi
-  - composer require typo3/cms="$TYPO3_VERSION"
+  - if [ "$TRAVIS_ALLOW_FAILURE" == "false" ] && [ "$TYPO3_VERSION" == "^6.2.0" ]; then composer require typo3/cms-composer-installers="1.2.5 as 1.1.4" --no-update; fi
```

Comment:
I would prefer to early return here is well if allow failure is enabled:
- if [ "$TRAVIS_ALLOW_FAILURE" == "true" ]; then echo "skip building"; return; fi;